BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -336,6 +370,15 @@ - (NSString *)fullBookmarkFieldsParameterString
     return [[self fullBookmarkFieldsArray] componentsJoinedByString:@","];
 }
 
+- (NSString *)performantItemFieldsParameterString
+{
+    NSMutableOrderedSet *set = [NSMutableOrderedSet orderedSet];
+    [set addObjectsFromArray:[self fullFolderFieldsArray]];
```

Comment:
With these full arrays in here we may actually accidentally reintroduce the non-performant fields, since the fields are generic and can be applied to any item, today folder and bookmark don't have 'BOXAPIObjectKeyVersionNumber' but if that was added tomorrow (may make sense for bookmark for example) and we added it to the full fields array, that would creep into this performant parameter string and we would have the same problem over again.